### DISCUSSION TITLE
SCREEN_07=menu

### DISCUSSION THREAD

**Discussion by aruansoares** [ltsp/ltsp#189] [Q&A]
Good afternoon. All right?

Today we currently use this setting:

SCREEN_07 = menu
MENU_ITEM_01 = "XCOMP125"
MENU_COMMAND_01 = "xfreerdp /v:192.168.0.125 / multimon / bpp: 32"
MENU_ITEM_02 = "XCOMP126"
MENU_COMMAND_02 = "xfreerdp /v:192.168.0.126 / multimon / bpp: 32"
MENU_ITEM_03 = "LDM"
MENU_COMMAND_03 = "LDM"

Accessing RDP and Ubuntu, depending on need. Is there a possibility today?

Thank you very much.

**Comment by alkisg** [comment]
Hi, you just need to create session.desktop files, and users can select them from the login screen like in this picture:
https://askubuntu.com/questions/81161/how-to-remove-session-entries-from-lightdm

I will move this issue to the community tracker as it doesn't require any changes in the LTSP codebase.
